src/content-distribution/distribute/style.scss

```diff
@@ -0,0 +1,53 @@
+.newspack-network-distribute {
+	height: calc(100% - 47px);
+	.components-panel {
+		height: 100%;
+		display: flex;
+		flex-direction: column;
+	}
+	.components-panel__body {
+		border: 0;
+		&.distribute-header {
+			p {
+				color: #757575;
```

Comment:
Instead of hardcoding colors, could we use WordPress color variables? https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_colors.scss#L11

Reply:
Yes! Updated in aa34cf4ce1a1d20fc124550e49849cf47978da31